ghre-rails/config/manifests/dev_manifest.yml

```diff
@@ -0,0 +1,12 @@
+---
+applications:
+- name: get-help-with-remote-education-rails-dev
+  disk_quota: 1G
+  memory: 1G
+  instances: 1
+  env:
+    RAILS_ENV: production
```

Comment:
shall we put `RAILS_SERVE_STATIC_FILES` in here as well?

Reply:
added RACK_ENV + RAILS_SERVE_STATIC_FILES